Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeItem.cs

```diff
@@ -6,7 +6,9 @@
 using Microsoft.Toolkit.Uwp.Extensions;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
 using Windows.UI.Xaml.Controls;
+using Microsoft.Toolkit.Uwp.UI.Automation.Peers;
 
 namespace Microsoft.Toolkit.Uwp.UI.Controls
```

Comment:
```suggestion
using Microsoft.Toolkit.Uwp.Extensions;
using Microsoft.Toolkit.Uwp.UI.Automation.Peers;
using Windows.UI.Xaml;
using Windows.UI.Xaml.Automation;
using Windows.UI.Xaml.Automation.Peers;
using Windows.UI.Xaml.Controls;

namespace Microsoft.Toolkit.Uwp.UI.Controls
```

Reply:
Ah thanks for that, hadn't noticed any errors when building the solution. I'll keep an eye on these in the future though 👍 Should be fixed now for both files